action.yml

```diff
@@ -0,0 +1,99 @@
+---
+name: Container Scanning Action
+author: Adfinis AG
+description: TODO
+inputs:
+  image-ref:
+    description: Image to scan
+    required: true
```

Comment:
it recently yelled at me and told me to define `type`. is `type` really optional in composite action inputs.

Reply:
it yelled at me and told me type doesn't exist

Reply:
it might be fully well that it's not needed, the docs are a bit confusing and while the re-useable workflow examples all want type, the composite docs and examples never seem to have a type :weary:

Reply:
i would've added them if my LSP agreed with it
![image](https://github.com/adfinis/container-scanning-action/assets/110528300/1bc17fc1-4bb0-4003-ba6e-563195e15f32)

Reply:
okay... so type is in fact probably, maybe, hopefully supported. let's give this a check and blame the docs if it's wrong.